lib/flipper/group.rb

```diff
@@ -0,0 +1,19 @@
+module Flipper
+  class Group
+    attr_reader :name
+
+    def initialize(name, &block)
+      name = name.to_sym
+      if name.to_s.include?(Types::Group::SEPARATOR)
```

Comment:
Hrm, I guess this could break old clients, if they happen to use this separator in group names.  That would be pretty surprising, though.